org.flightgear.FlightGear.json

```diff
@@ -0,0 +1,191 @@
+			"config-opts": [
+				"--disable-everything",
+				"--disable-static",
+				"--enable-shared",
+				"--disable-programs",
+				"--disable-doc",
+				"--disable-avdevice",
+				"--disable-avfilter",
+				"--disable-postproc",
+
+				/* Enough to run the base game, might need extending for mod content */
+				"--enable-decoder=bink,binkaudio_rdft,mp3,pcm_s16le,vorbis",
+				"--enable-demuxer=bink,mp3,ogg,pcm_s16le,wav"
+			],
+			"sources": [
+				{
+				"type": "archive",
+				"url": "http://ffmpeg.org/releases/ffmpeg-2.8.14.tar.xz",
+				"sha256": "ef061d9a6cdde8628d00613d17b467fb5ad75e7404ef523dc842e192c511a116"
+				}
+			]
+		},
+		{
+			"name": "openscenegraph",
+			"buildsystem": "cmake-ninja",
+			"config-opts": [
+				"-DCMAKE_BUILD_TYPE=Release"
+			],
+			"sources": [
+				{
+					"type":"archive",
+					"url":"https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-3.2.1.tar.gz",
+					"sha256":"2083b632b764438727e07289845992d41757582e571e3133ca259a6514cf72b2"
+				},
+				{
+					"type": "shell",
+					"commands": [
+						"sed 's|SET(LIB_POSTFIX \"64\" CACHE|SET(LIB_POSTFIX \"\" CACHE|' -i CMakeLists.txt",
+						"sed 's|DGifCloseFile(giffile)|DGifCloseFile(giffile,0)|' -i src/osgPlugins/gif/ReaderWriterGIF.cpp"
+					]
+				}
+			]
+		},
+		{
+			"name": "simgear",
+			"buildsystem": "cmake-ninja",
+			"config-opts": [
+				"-DCMAKE_BUILD_TYPE=Release"
+			],
+			"sources": [
+				{
+					"type":"archive",
+					"url":"https://sourceforge.net/projects/flightgear/files/release-2017.3/simgear-2017.3.1.tar.bz2",
+					"sha256":"0ee08550b737b249dcc91590ec0cb9c5dc9080998f6ba66a7d7209cdfce6e1f4"
+				}
+			]
+		},
+		{
+			"name": "xmu",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.x.org/archive//individual/lib/libXmu-1.1.2.tar.bz2",
+					"sha256": "756edc7c383254eef8b4e1b733c3bf1dc061b523c9f9833ac7058378b8349d0b"
+				}
+			]
+		},
+		{
+			"name": "plib",
+			"sources": [
+				{
+					"type": "archive",
+					"url": "http://plib.sourceforge.net/dist/plib-1.8.5.tar.gz",
+					"sha256": "485b22bf6fdc0da067e34ead5e26f002b76326f6371e2ae006415dea6a380a32"
+				}
+			]
+		},
+		{
+			"name": "flightgear",
+			"buildsystem": "cmake-ninja",
+			"config-opts": [
+				/*"-DENABLE_QT=ON",*/
+				"-DCMAKE_BUILD_TYPE=Release",
+				"-DFG_BUILD_TYPE=Release"
+			],
+			"builddir": true,
+			"sources": [
+				{
+					"type":"archive",
+					"url":"https://sourceforge.net/projects/flightgear/files/release-2017.3/flightgear-2017.3.1.tar.bz2",
+					"sha256":"6f2e1d992e2f202b8f9c918c9fb19124ef06824ea0e767e2f4dff6ba43728ccd"
+				},
+				{
+					"type": "file",
+					"path": "org.flightgear.FlightGear.appdata.xml"
+				},
+				{
+					"type": "file",
+					"path": "flightgear.sh"
+				}
+			],
+			"build-commands": [
+				"install -Dm644 ../org.flightgear.FlightGear.appdata.xml /app/share/appdata/org.flightgear.FlightGear.appdata.xml",
+				"install -Dm744 ../flightgear.sh /app/bin/flightgear.sh"
```

Comment:
You probably need to patch the desktop file to use the wrapper script?

Reply:
Right, done.